docs/languages/main-trio.md

```diff
@@ -1,5 +1,5 @@
-# 🔧 BCL API reference documentation
+# 🔧 The main trio: C#, F# and VB.NET
```

Comment:
Should we call it VB.NET or VB?  @richlander suggested that I change VB.NET to VB in the async docs a while back.